common/src/proto.rs

```diff
@@ -81,4 +82,5 @@ pub struct FreshNode {
 pub struct Character {
     pub name: String,
     pub orientation: na::UnitQuaternion<f32>,
+    pub velocity: na::Vector3<f32>,
```

Comment:
These fields should be replaced with a single `CharacterState` field, since otherwise we're just manually inlining it.